app/models/sample.rb

```diff
@@ -25,7 +25,7 @@ class Sample < ApplicationRecord
   has_many :workflow_executions, through: :samples_workflow_executions
 
   validates :name, presence: true, length: { minimum: 3, maximum: 255 }
-  validates :name, uniqueness: { scope: %i[name project_id] }
+  validates :name, uniqueness: { scope: %i[project_id] }
```

Comment:
**nitpick (blocking):** you can remove the `%i[project_id]` and just replace with `:project_id`